mobile/lib/features/trade/domain/api_response.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:get_10101/features/trade/domain/response_status.dart';
```

Comment:
Fair enough - tbh I have not invested much thought into this model; not sure it's such a great idea to use it in that way. We could also stick to exceptions here instead of creating artificial types. 
I'll give it another thought

Reply:
removed these classes in favor of exception handling in the change notifier